react/components/RadioGroup/index.js

```diff
@@ -44,8 +47,8 @@ class RadioGroup extends React.Component {
             const id = `${name}-${i}`
             return (
               <label
-                className={`db br3 ${classNames({
-                  'b--muted-4 ba pv2 ph4': !hideBorder,
+                className={`db br3 ${borderColor} ${classNames({
+                  'ba pv2 ph4': !hideBorder,
                   pointer: !isDisabled,
                 })}`}
```

Comment:
```suggestion
                className={classNames('db br3', {
                  'b--danger hover-b--danger': error || errorMessage,
                  'b--muted-4': !error && !errorMessage,
                  'ba pv2 ph4': !hideBorder,
                  pointer: !isDisabled,
                })}
```

Also there could be a variable `const errorHighlight = error || errorMessage` to make it cleaner

Reply:
Nice. I have made the changes ✨